### PR TITLE
Fix: fix check in get method

### PR DESCRIPTION
**Pull request by sillyElvy**
## About

This pull request is about (X), which does (Y).

## Checklist

- [x] I've ran `pre-commit` to format and lint the change(s) made.
- [ ] I've checked to make sure the change(s) work on `3.8.6` and higher.
- [ ] This fixes/solves an [Issue](https://github.com/goverfl0w/discord-interactions/issues).
  - (If existent):
- I've made this pull request for/as: (check all that apply)
  - [ ] Documentation
  - [ ] Breaking change
  - [ ] New feature/enhancement
  - [x] Bugfix

### CODE REVIEW

**Review by Damego — APPROVED** [review]

File: interactions/client/get.py [hunk 1]
```diff
@@ -169,7 +169,11 @@ def get(client: Client, obj: Type[_T], **kwargs) -> Optional[_T]:
     if version_info >= (3, 9):
 
         def _check():
-            return obj == list[get_args(obj)[0]]
+            return (
+                obj == list[get_args(obj)[0]]
```
Comment (Toricane) [inline interactions/client/get.py:173]: What is `list`?

Comment (sillyElvy) [reply interactions/client/get.py:173]: Builtin. Works for typing since 3.9

Comment (Toricane) [reply interactions/client/get.py:173]: Aaaand what is the minimum version of python for this library?

Comment (Toricane) [reply interactions/client/get.py:173]: Nevermind

**Review by Toricane — APPROVED** [review]
LGTM!

**Pull request merged**